okhttp/src/test/java/okhttp3/internal/tls/HostnameVerifierTest.java

```diff
@@ -373,7 +380,11 @@
         + "-----END CERTIFICATE-----\n");
 
     X509Certificate peerCertificate = ((X509Certificate) session.getPeerCertificates()[0]);
-    assertThat(certificateSANs(peerCertificate)).containsExactly("*.bar.com", "*.������.co.jp");
+    if (platform.isConscrypt()) {
+      assertThat(certificateSANs(peerCertificate)).containsExactly("*.bar.com");
+    } else {
+      assertThat(certificateSANs(peerCertificate)).containsExactly("*.bar.com", "*.������.co.jp");
```

Comment:
is this asserting a bug? Shouldn’t the cert library be limiting these to ASCII strings?

Reply:
(the bug is in JSSE, Coretto, etc.)

Reply:
Yep.